# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------
joint1_position_controller:
  type: "position_controllers/JointPositionController"
  joint: joint1

joint2_position_controller:
  type: "position_controllers/JointPositionController"
  joint: joint2

joint3_position_controller:
  type: "position_controllers/JointPositionController"
  joint: joint3

joint6_position_controller:
  type: "position_controllers/JointPositionController"
  joint: joint6

joint8_position_controller:
  type: "position_controllers/JointPositionController"
  joint: joint8

gazebo_ros_control/pid_gains:
  joint1: {p: 1, i:  1, d:  0.1}
  joint2: {p: 2, i:   0, d: 0.1}
  joint3: {p: 1, i: 0, d: 0.1}
  joint6: {p: 1.5, i: 0, d: 0.1}
  joint8: {p: 1.5, i:  0.0, d: 0.1}

joint_limits:
  joint1:
    has_velocity_limits: true
    max_velocity: 0.5
  joint2:
    has_velocity_limits: true
    max_velocity: 0.5
  joint3:
    has_velocity_limits: true
    max_velocity: 0.5
  joint4:
    has_velocity_limits: true
    max_velocity: 0.5
  joint5:
    has_velocity_limits: true
    max_velocity: 0.5
  joint6:
    has_velocity_limits: true
    max_velocity: 0.5
  joint7:
    has_velocity_limits: true
    max_velocity: 0.5
  joint8:
    has_velocity_limits: true
    max_velocity: 0.5
